modules/lib/src/main/resources/assets/js/app/wizard/WorkflowStateManager.ts

```diff
@@ -94,10 +94,22 @@ export class WorkflowStateIconsManager {
         };
     }
 
-    static toggleWorkflowStateClasses(element: Element, status: WorkflowStateStatus) {
-        element.toggleClass(WorkflowStateIconsManager.INVALID_CLASS, status.invalid);
-        element.toggleClass(WorkflowStateIconsManager.READY_CLASS, status.ready);
-        element.toggleClass(WorkflowStateIconsManager.IN_PROGRESS_CLASS, status.inProgress);
+    private toggleWorkflowStateClasses(element: Element) {
+        element.toggleClass(WorkflowStateManager.INVALID_CLASS, this.status.invalid);
+        element.toggleClass(WorkflowStateManager.READY_CLASS, this.canMarkIconElementAsReady());
+        element.toggleClass(WorkflowStateManager.IN_PROGRESS_CLASS, this.status.inProgress);
+    }
+
+    private canMarkIconElementAsReady(): boolean {
```

Comment:
I don't think this is a correct method name (hard to understand, too). Should be similar to `isPublished` and `isMoved` - `isReady` or `isInReadyWorkflowState` or `isMarkedAsReady`.

Reply:
Done